troposphere/__init__.py

```diff
@@ -525,25 +525,33 @@ def __init__(self, data):
         self.data = {'Fn::ImportValue': data}
 
 
+class Tag(AWSHelperFn):
```

Comment:
My intention was to handle the Tag formatting for values in my If function.  Basically, I wanted to be lazy and write `Tag('key', 'value')` instead of `{'Key': 'key', 'Value': 'value'}` as the True and False values for my If statement.  I have an example in the test_TagConditional` method.

I tried initially to reuse the Tags class, however that resulted in each value being a list instead of just the single tag.  Basically, this:

```
Tags(
    {'foo': 'foo'},
    If('MyCondition', Tags('bar', 'bar'), Tags('baz', 'baz'))
)
```

resulted in 

```
[
  {
    "Fn::If": [
       "MyCondition",
       [{"Key": "bar", "Value": "bar"}]
       [{"Key": "baz", "Value": "baz"}]
    ]
  },
  'Value': 'foo', 'Key': 'foo'}
]
```

Just setting the data value seemed simpler than wrapping\extending the Tags class and then writing some logic to only convert the first value in the underlying list, but I can update to that method if desired.

Reply:
Got it. Thanks for the explanation.